browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -190,6 +190,60 @@ class MarkdownNoteDetail extends React.Component {
     ee.emit('export:save-html')
   }
 
+  handleKeyDown (e) {
+    switch (e.keyCode) {
+      // tab key
+      case 9:
+        if (e.ctrlKey && !e.shiftKey) {
+          e.preventDefault()
+          this.jumpNextTab()
+        } else if (e.ctrlKey && e.shiftKey) {
+          e.preventDefault()
+          this.jumpPrevTab()
+        } else if (!e.ctrlKey && !e.shiftKey && e.target === this.refs.description) {
+          e.preventDefault()
+          this.focusEditor()
+        }
+        break
+      // I key
+      case 73:
+        {
+          const isSuper = global.process.platform === 'darwin'
+            ? e.metaKey
+            : e.ctrlKey
+          if (isSuper) {
+            e.preventDefault()
+            this.handleInfoButtonClick(e)
+          }
+        }
+        break
+      // L key
+      case 76:
+        {
+          const isSuper = global.process.platform === 'darwin'
+            ? e.metaKey
+            : e.ctrlKey
+          if (isSuper) {
+            e.preventDefault()
+            this.focus()
+          }
+        }
+        break
+      // T key
+      case 84:
+        {
+          const isSuper = global.process.platform === 'darwin'
+            ? e.metaKey
+            : e.ctrlKey
+          if (isSuper && !e.shiftKey && !e.altKey) {
+            e.preventDefault()
+            this.addSnippet()
```

Comment:
What's this `addSnippet` function :smile:

Reply:
Ok. 
My bad
I thought, since I was at it, I might as well add the shortcuts already implemented for the SnippetNode.
I'll push again with only ctrl+i and ctrl+t

Reply:
i mean ctrl+i and ctrl+tab